gossip/src/crds_gossip_push.rs

```diff
@@ -282,6 +282,10 @@ impl CrdsGossipPush {
             stakes,
         )
     }
+
+    pub(crate) fn is_push_active_set_initialized(&self) -> bool {
+        !self.active_set.read().unwrap().is_empty()
+    }
```

Comment:
also mentioned in the chat, but this is pretty deep in the implementation detail,
and i would rather not rely on this or expose it in any form of api.
in particular this will make future changes to gossip pretty difficult